libminifi/src/utils/StringUtils.cpp

```diff
@@ -305,8 +305,10 @@ bool StringUtils::from_base64(uint8_t* data, size_t* data_length, const char* ba
     case 1:
       return false;
     case 2:
-      digits[2] = 0x00;
     case 3: {
+      if (digit_counter == 2) {
```

Comment:
You probably did not mean to do this.

Reply:
After further checking the issue clang does not include the implicit-fallthrough warning in Wall and Wextra flags, so it is only GCC specific issue. To be backward compatible with older GCC versions I used `// fall through` marker comment.  Updated in 9b337b70b01441aea445a5d31a75c88c7ab6f680.